app/controllers/admin/merchants_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Admin::MerchantsController < ApplicationController
```

Comment:
We should be inheriting from `Admin::BaseController` since this performs the before_action to check to see if the user is a site admin.